appstudio-controller/controllers/appstudio.redhat.com/promotionrun_controller.go

```diff
@@ -233,7 +232,6 @@ func (r *PromotionRunReconciler) Reconcile(ctx context.Context, req ctrl.Request
 			return ctrl.Result{}, fmt.Errorf("unable to update Binding '%s' snapshot: %v", binding.Name, err)
 		}
 
-		logutil.LogAPIResourceChangeEvent(promotionRun.Namespace, promotionRun.Name, promotionRun, logutil.ResourceModified, log)
```

Comment:
I think this should _should_ exist, but should instead be a ResourceModified on the binding, due to the `binding.Spec.Snapshot` update above.